prediction_market_agent/agents/autogen_general_agent/deploy.py

```diff
@@ -0,0 +1,62 @@
+from datetime import timedelta
+
+from loguru import logger
+from prediction_market_agent_tooling.config import PrivateCredentials
+from prediction_market_agent_tooling.deploy.agent import DeployableTraderAgent
+from prediction_market_agent_tooling.markets.data_models import Bet
+from prediction_market_agent_tooling.markets.markets import MarketType
+from prediction_market_agent_tooling.tools.utils import utcnow
+
+from prediction_market_agent.agents.autogen_general_agent.social_agent import (
+    build_social_media_text,
+)
+from prediction_market_agent.agents.autogen_general_agent.social_media.abstract_handler import (
+    AbstractSocialMediaHandler,
+)
+from prediction_market_agent.agents.autogen_general_agent.social_media.farcaster_handler import (
+    FarcasterHandler,
+)
+from prediction_market_agent.agents.autogen_general_agent.social_media.twitter_handler import (
+    TwitterHandler,
+)
+from prediction_market_agent.utils import APIKeys
+
+
+class DeployableSocialMediaAgent(DeployableTraderAgent):
+    model: str = "gpt-4-turbo-2024-04-09"
+    social_media_handlers: list[AbstractSocialMediaHandler] = [
+        FarcasterHandler(),
+        TwitterHandler(),
+    ]
+
+    def run(self, market_type: MarketType) -> None:
+        # It should post a message (cast) on each run.
+        # We just need one market to get latest bets.
```

Comment:
Why do we need one market to get the latest bets?

Reply:
Think you were looking at an older version of this PR 😃 
<img width="819" alt="Screenshot 2024-05-13 at 11 15 47" src="https://github.com/gnosis/prediction-market-agent/assets/56087052/2fc321f6-5dc3-4be4-9ef6-e9610a5b25bd">

Reply:
That comment landed in `main` branch https://github.com/gnosis/prediction-market-agent/blob/main/prediction_market_agent/agents/autogen_general_agent/deploy.py#L34 😄 

So I guess it's just a forgotten comment: https://github.com/gnosis/prediction-market-agent/pull/164 🙏